package.json

```diff
@@ -82,13 +94,34 @@
         ],
         "extensions": [
           ".tasty-decoded"
-        ]
+        ],
+        "icon": {
+          "dark": "icons/file_type_scala.svg",
+          "light": "icons/file_type_scala.svg"
+        }
       },
       {
         "id": "buildtarget",
         "extensions": [
           ".metals-buildtarget"
-        ]
+        ],
+        "icon": {
+          "dark": "icons/book-dark.svg",
```

Comment:
Is it a default one for vscode?
I don't see it in pr

Reply:
It was actually already added previously, I reused an existing icon.